packages/dds/tree/src/tableSchema.ts

```diff
@@ -483,280 +328,74 @@ export namespace TableSchema {
 		TScope extends string | undefined = string | undefined,
 		TCellSchema extends ImplicitAllowedTypes = ImplicitAllowedTypes,
 		TPropsSchema extends ImplicitFieldSchema = ImplicitFieldSchema,
-	> = ReturnType<typeof createRow<TScope, TCellSchema, TPropsSchema>>;
+	> = ReturnType<typeof TableSchema.createRow<TScope, TCellSchema, TPropsSchema>>;
 
 	// #endregion
 
 	// #region Table
 
 	/**
-	 * A key to uniquely identify a cell in a table.
-	 * @sealed @internal
+	 * Factory for creating new table schema.
+	 * @system @internal
 	 */
-	export interface CellKey {
-		/**
-		 * {@link TableSchema.IColumn.id} of the containing {@link TableSchema.IColumn}.
-		 */
-		readonly columnId: string;
+	// eslint-disable-next-line @typescript-eslint/explicit-function-return-type -- Return type is too complex to be reasonable to specify
+	export function createTableInternal<
```

Comment:
The "Internal" postfix on these factory functions is probably no longer needed. And if the containing namespace is strictly type-exported, there's no risk of them being consumed outside of this module.